swimlane/core/client.py

```diff
@@ -246,14 +263,50 @@ def user(self):
         """User record instance for authenticated user"""
         return self._session.auth.user
 
+class SwimlaneTokenAuth(SwimlaneResolver):
+    """Handles token authentication for all requests"""
+
+    def __init__(self, swimlane, access_token):
+        super(SwimlaneTokenAuth, self).__init__(swimlane)
+
+        self._access_token = access_token
+        self.user = None
+    
+    def __call__(self, request):
+        """Attach necessary headers to all requests"""
+
+        headers = {
+            'Private-Token': self._access_token
+        }
+
+        request.headers.update(headers)
+
+        # Only make the call to user/authorize to get the user's profile if we haven't retrieved it
+        # already
+        if self.user != None:
```

Comment:
Not an issue here, but minor python point. Almost always want `if self.user is not None` vs `!=` in cases like this where it's used as a sentinel value. Anything `falsey` will match in that case, so `0`, `True`, etc. Fine here since it would (should) only ever be a `User` instance.

Reply:
Always good to know, went ahead and changed it to keep things consistent throughout the file